wrappers/winnt.go

```diff
@@ -1209,6 +1209,16 @@ const (
 	JobObjectGroupInformation                   = 11
 )
 
+const (
+	EVENT_ALL_ACCESS   = 0x1F0003
+	EVENT_MODIFY_STATE = 0x0002
```

Comment:
Let's define `EVENT_ALL_ACCESS` as `STANDARD_RIGHTS_REQUIRED | SYNCHRONIZE | 0x0003` to match what's in winnt.h.  Also, the order of these constants is flipped relative to what's in winnt.h.